SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -289,7 +289,8 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 			FString URLString = FURL().ToString();
 			URLString += TEXT("?workerAttribute=") + UnrealMetadataComponent->OwnerWorkerAttribute;
 
-			Connection = NetDriver->AcceptNewPlayer(FURL(nullptr, *URLString, TRAVEL_Absolute), true);
+			// TODO: Once we can checkout PlayerController and PlayerState atomically, we can grab the UniqueId and online subsystem type from PlayerState.
```

Comment:
The worker team is currently thinking about a solution to this problem (essentially deleting critical sections and having something better).